ansible_collections/arista/avd/plugins/action/eos_designs_facts.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import cProfile
+import pstats
+from ansible.errors import AnsibleActionFail
+from ansible.plugins.action import ActionBase
+from ansible.plugins.filter.core import combine
+from ansible.plugins.loader import lookup_loader
+from ansible.template import Templar
+from ansible_collections.arista.avd.plugins.filter.convert_dicts import convert_dicts
+from ansible_collections.arista.avd.plugins.filter.list_compress import list_compress
+from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
+from ansible_collections.arista.avd.plugins.module_utils.eos_designs_facts import EosDesignsFacts
+from ansible_collections.arista.avd.plugins.module_utils.utils import AristaAvdMissingVariableError
+
+
+class ActionModule(ActionBase):
+    def run(self, tmp=None, task_vars=None):
+        if task_vars is None:
+            task_vars = {}
```

Comment:
```
    def run(self, tmp=None, task_vars={}):
```

I believe we can write it this way.

Reply:
Yes, thank you for clarifying this. I found this article: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python